src/TestFramework/PhpUnit/Coverage/IndexXmlCoverageParser.php

```diff
@@ -65,7 +69,9 @@ public function parse(string $coverageIndexPath, string $xmlIndexCoverageContent
     {
         $xPath = XPathFactory::createXPath($xmlIndexCoverageContent);
 
-        self::assertHasCoverage($xPath);
+        $totalCoverage = self::retrieveTotalCoverage($xPath);
+
+        $this->eventDispatcher->dispatch(new TotalLineCodeCoverageWasCalculated($totalCoverage));
```

Comment:
I cannot think of a better approach so I guess this'll go. I'm doing something very similar in #1159 FWIW.

Reply:
in what?

Reply:
Ah #1153